README.md

```diff
@@ -2,10 +2,32 @@
 
 # App Scaffold
 
-_Note: the App Scaffold currently depends on zat v1.35.12 or greater._
```

Comment:
It's only relevant when running locally. I don't think it should be the first thing people read.